### DISCUSSION TITLE
Lazyflex.nvim: Makes it easier to test and troubleshoot a LazyVim configuration.

### DISCUSSION THREAD

**Discussion by abeldekat** [LazyVim/LazyVim#1780] [Show and tell]
An excerpt from the [README](https://github.com/abeldekat/lazyflex.nvim):

# lazyflex.nvim

**lazyflex.nvim** is an add-on for [**lazy.nvim**](https://github.com/folke/lazy.nvim), a modern plugin manager for Neovim.

Its main objective is to make it easier to test and troubleshoot a `neovim` configuration.

## Demo

https://github.com/abeldekat/lazyflex.nvim/assets/58370433/1bb1ba01-b6a3-4753-b3d7-e05bc1cd1ce7

> The code used in the demo can be found [here](https://github.com/abeldekat/starter/blob/lazyflex_demo/lua/config/lazyflex_test.lua)

## Features

- Easier troubleshooting/testing from one central location.
  - Enable/disable multiple plugins by keyword.
  - Define and use **presets** for your own configuration.
  - Has **presets** for each default plugin module in [**LazyVim**](https://github.com/LazyVim/LazyVim).
  - Has options to skip loading the configuration modules (`options`, `autocmds`, `keymaps`) provided by **LazyVim**.
- Helps to verify the independence of the components in the configuration.
- When creating an issue, facilitates writing a concise reproducible configuration.
  - Contains examples for minimal configurations using **lazyflex**.

**Comment by dpetka2001** [comment]
Really useful plugin for quickly testing with disabling and only enabling what you want to bisect.
